framework/assets/yii.js

```diff
@@ -144,8 +144,8 @@ yii = (function ($) {
          * @param $e the jQuery representation of the element
          */
         handleAction: function ($e, event) {
-            var method = $e.data('method'),
-                $form = $e.closest('form'),
+            var $form = $e.attr('data-form') ? $('#' + $e.attr('data-form')) : $e.closest('form'),
```

Comment:
@SilverFire Another con of `.data()` is the fact that it binds people to `jQuery`.
Basically any non-jquery javascript that changes the attribute will not work after the first form submission. Since these form submissions are likely not to cause a page refresh this will cause unexpected behavior.
Developers should be free to not using jQuery in their own javascript files regardless if Yii uses it in its core.